partials/footer.hbs

```diff
@@ -23,98 +23,88 @@
-                            Map</a>
+                            href="https://www.freecodecamp.org/news/javascript-es6-promises-for-beginners-resolve-reject-and-chaining-explained/">JavaScript
+                            Promise</a>
                         <a
-                            href="https://www.freecodecamp.org/news/what-is-https-a-guide-to-secure-web-browsing-and-browser-encryption/">What
-                            is HTTPS?</a>
-
-                        <a href="https://www.freecodecamp.org/news/python-list-append-vs-python-list-extend/">Python
-                            List Append</a>
+                            href="https://www.freecodecamp.org/news/what-is-github-what-is-git-and-how-to-use-these-developer-tools/">What
+                            is GitHub?</a>
                         <a
-                            href="https://www.freecodecamp.org/news/what-is-chromium-how-the-chromium-web-browser-is-different-from-chrome/">What
-                            is Chromium?</a>
-                        <a href="https://www.freecodecamp.org/news/smoke-testing/">Smoke Testing</a>
-
+                            href="https://www.freecodecamp.org/news/the-python-sort-list-array-method-ascending-and-descending-explained-with-examples/">Python
+                            Sort List</a>
+                        <a href="https://www.freecodecamp.org/news/json-comment-example-how-to-comment-in-json-files/">Comments
+                            in JSON</a>
+                        <a
+                            href="https://www.freecodecamp.org/news/what-is-kanban-the-agile-methodology-defined-and-how-to-use-it-for-your-software-development-team-2/">What
+                            is Kanban?</a>
                         <a
-                            href="https://www.freecodecamp.org/news/clear-history-how-to-delete-your-browsing-history-in-chrome-firefox-and-safari/">Clear
-                            History</a>
+                            href="https://www.freecodecamp.org/news/python-write-to-file-open-read-append-and-other-file-handling-functions-explained/">Python
+                            Write to File</a>
                         <a
-                            href="https://www.freecodecamp.org/news/private-browser-how-to-use-incognito-mode-in-chrome-and-safari/">Incognito
-                            Mode</a>
+                            href="https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/">CSS
+                            Media Queries</a>
                         <a
-                            href="https://www.freecodecamp.org/news/linux-how-to-add-users-and-create-users-with-useradd/">Linux
-                            Add User</a>
+                            href="https://www.freecodecamp.org/news/clear-history-how-to-delete-your-browsing-history-in-chrome-firefox-and-safari/">HTML
+                            Entities</a>
```

Comment:
This text is a bit different than the actual article. Perhaps it could read:

<a href="https://www.freecodecamp.org/news/clear-history-how-to-delete-your-browsing-history-in-chrome-firefox-and-safari/">Clear Browser History</a>

Reply:
It should be resolved